src/UI/Component/Input/Field/Factory.php

```diff
@@ -205,4 +206,59 @@ public function dependantGroup(array $inputs);
 	 * @return    \ILIAS\UI\Component\Input\Field\Checkbox
 	 */
 	public function checkbox($label, $byline = null);
+
+
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *     A TagInput is used to choose a subset amount of tags out of a finite list
+	 *     of tags. The TagInput is used whenever not all available tags to choose from
+	 *     MUST or SHOULD be visible, e.g. because the amount is too high.
+	 *     Besides the tags to choose from, the user can provide own tags by typing them
+	 *     into the Input
+	 *   composition: >
+	 *     The Input is presented as a TextInput and prepended by already selected tags
+	 *     presented as texts including a Close-Button.  (e.g. [ Amsterdam X ] )
+	 *     The input is labeled by the $label given.
```

Comment:
I Do not understand "The input is labeled by the $label given." 
Does that mean "Tags bear the label users have provided once they were saved."?

Reply:
Get rid of the $ in the description.